backend/src/services/token/token.service.ts

```diff
@@ -9,25 +15,27 @@ type Constructor = {
 class Token {
   #alg: string;
   #expiresIn: string;
+  #secretKey: Promise<KeyLike | Uint8Array>;
```

Comment:
```suggestion
  #secret: string;
```
generate by `generateSecret` only in places where you need it

Reply:
Im little bit confused what key i should use to verify token